### PR TITLE
feat: --keepEmptySchema option

### PR DESCRIPTION
**Pull request by mamatsunami**
**Context**

When filtering a OpenAPI spec, openapi-format automatically remove empty objects. I had [some issues with the openapi-generator because it doesn't handle empty request body](https://github.com/OpenAPITools/openapi-generator/issues/9217).

I wanted to format a OpenAPI spec generated by Symfony API Platform which had some empty schema response object like this :
```yaml
requestBody:
  description: Create a new pet in the store
  required: true
  content:
    application/json:
      schema: {}
```

It was converting them to this :
```yaml
requestBody:
  description: Create a new pet in the store
  required: true
  content:
    application/json: {}
```

Which causes a `java.lang.RuntimeException: Request body cannot be null` with openapi-generator when I try to generate an API client from the filtered spec file.

**Solution**

I added a `--keepEmptySchema` parameter which disable the removal of empty schema object as a workaround.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -157,6 +158,7 @@ Options:
 | --filterFile         | -f            | the file to specify filter setting                                          | path to file | defaultFilter.json          | optional  |
 | --casingFile         | -c            | the file to specify casing setting                                          | path to file |                             | optional  |
 | --no-sort            |               | don't sort the OpenAPI file                                                 | boolean      | FALSE                       | optional  |
+| --keepEmptySchema    |               | don't remove empty schema object                                            | boolean      | FALSE                      | optional |
```
Comment (thim81) [inline readme.md:161]: very minor but could you align the table spacing at the end. It seems it is missing a space before "| optional |"

**Review by thim81 — CHANGES_REQUESTED** [review]
Looks good.

File: readme.md (hunk 1 above)
Comment (mamatsunami) [reply readme.md]: nice catch ! it's fixed ;-)

**Review by thim81 — APPROVED** [review]

**Comment by mamatsunami** [comment]
@thim81 Just realizing there is other case where it should be nice to prevent the removal of empty objects.
I've seen you didn't yet create a release from my commit.

I was thinking to rework my current `--keepEmptySchema` option to make it generic. I think it should be more flexible to rename it `--keepEmptyObjects`. `--keepEmptyObjects` would skip the removal of all empty object, but you could also pass an array of property to only prevent the removal of empty object in some cases, for instance `--keepEmptyObjects=examples,schema`.

Can you wait I submit another PR before publishing a new version ?

**Comment by thim81** [comment]
That would be a more generic approach, so looking forward for you PR.

**Comment by thim81** [comment]
Could you also check that the new `--keepEmptyObjects` is also supported through the `--configFile` option? 
Since it offers more options that just a simple boolean, so it would be handy to define this in the JSON config file that can be passed along.

**Comment by mamatsunami** [comment]
@thim81 here it is ! https://github.com/thim81/openapi-format/pull/85

**Pull request merged**